tests/unit/eia_archiver_test.py

```diff
@@ -50,7 +50,7 @@ async def test_eia860m(mocker):
     )
     archiver = Eia860MArchiver(mock_session, only_years=[2019, 2022])
     resources = [res async for res in archiver.get_resources()]
-    assert len(resources) == 24
+    assert len(resources) == 2
```

Comment:
Do we want to / can we assert anything about the contents/structure of these resources (e.g. that the number of files they contain == 12?)

Reply:
Just added some mocks to do this!